lib/utils/cache.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+const LRUMap = require('lru-cache');
```

Comment:
https://www.npmjs.com/package/lru-cache

So this cache is just held in memory while the program is running, or is it persisted somewhere? Any reason why we're not using Redis as initially discussed? Happy either way, just interested :)